node/service/src/chain_spec.rs

```diff
@@ -157,7 +157,24 @@ pub fn rococo_config() -> Result<RococoChainSpec, String> {
 }
 
 pub fn versi_config() -> Result<VersiChainSpec, String> {
-	VersiChainSpec::from_json_bytes(&include_bytes!("../res/versi.json")[..])
+	let wasm_binary = rococo::WASM_BINARY.ok_or("Versi development wasm not available")?;
+
+	Ok(RococoChainSpec::from_genesis(
+		"Versi",
+		"versi_v1_3",
+		ChainType::Live,
+		move || RococoGenesisExt {
+			runtime_genesis_config: rococo_development_config_genesis(wasm_binary),
+			// Use 1 minute session length.
+			session_length_in_blocks: Some(100),
+		},
+		vec![],
```

Comment:
Bootnodes are being set to null here, which I think is alright since we have our own spec for deployments in ansible:

https://github.com/paritytech/substrate/blob/master/client/chain-spec/src/chain_spec.rs#L263

Reply:
Let's just kick them above then.

Reply:
I would go even further to suggest that we should remove the versi.json entirely.

It would be our responsibility to set all the bootnodes and telemetry values in our ansible scripts.